test/spec/exception-handling.wast

```diff
@@ -2,38 +2,110 @@
   (event $e0 (attr 0) (param i32))
   (event $e1 (attr 0) (param i32 f32))
 
-  (func (export "throw_test")
-    (throw $e0 (i32.const 0))
+  (func (export "throw_single_value")
+    (throw $e0 (i32.const 5))
   )
 
-  (func (export "rethrow_test")
+  (func (export "throw_multiple_values")
+    (throw $e1 (i32.const 3) (f32.const 3.5))
+  )
+
+  (func (export "rethrow_nullref")
     (rethrow (ref.null))
   )
 
-  (func (export "try_test") (result i32)
+  (func (export "try_nothrow") (result i32)
     (try (result i32)
```

Comment:
Not for this PR, but I wonder if it would make sense to have syntax like 

```
(try
  (then ...)
  (catch ...)
)
```

To match the if-then-else syntax.

Reply:
Hmm, not sure, because the try body is always executed, so it isn't something that depends on some condition, like `then` clause in `if`.

Reply:
Rossberg was arguing for this in https://github.com/WebAssembly/exception-handling/issues/52 too. I think it makes sense, since the `catch` isn't actually nested within the try block.

Reply:
You mean, making it mandatory or optional? `if`s `then` and `else` are not mandatory.

Reply:
If optional, `do` might make sense, as Rossberg suggested.

Reply:
I think we should make it mandatory. `then` and `else` used to be optional (and I think still are in wabt), but the spec requires them now (see https://webassembly.github.io/spec/core/text/instructions.html#folded-instructions)

Reply:
I see. Yeah, it makes sense. I reopened WebAssembly/exception-handling#52. I think this PR can land without that and we can make this change separately, if we decide to do that.